main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+)
+
+func main() {
+	buf := bufio.NewReader(os.Stdin)
+	newContent := [][]byte{}
+
+	for {
+		content, readErr := buf.ReadBytes('\n')
+		if readErr != nil && readErr != io.EOF {
+			fmt.Fprintln(os.Stderr, readErr)
+			os.Exit(1)
+		}
+
+		if len(content) <= 0 {
+			return
+		}
+
+		trimmedContent := bytes.Trim(content, "\n")
+		newContent = append(newContent, trimmedContent)
```

Comment:
You're still collecting all data in memory.

Reply:
you want me to proccess the data one by one? How am I supposed to to that???

Reply:
By remembering the last printed line and comparing with it.

Reply:
that sounds way more easy than the way i did ._.